packages/chain-adapters/src/utxo/UtxoBaseAdapter.ts

```diff
@@ -402,6 +402,20 @@ export abstract class UtxoBaseAdapter<T extends UtxoChainId> implements IChainAd
     }
   }
 
+  async signAndBroadcastTransaction(signTxInput: SignTxInput<SignTx<T>>): Promise<string> {
```

Comment:
Urghhh, this might be why all tests are failing in super weird ways - we have many checks for `adapter.signAndBroadcastTransaction`

Reply:
Disregard me - went through some rambo debugging starting from clean origin/develop and running `git checkout origin/feat_snaps_btn -- filePath` adding files one by one and running `yarn && yarn build:packages && yarn && yarn test` every time until things break, and this is *not* the cause of the failing send tests